src/pages/Welcome/style.css

```diff
@@ -1,18 +1,81 @@
+.welcome-container {
+  align-items: center;
+  background-color: #f2b807;
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  width: 100%;
+}
+
+.welcome-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
 .welcome-form {
-  width: 250px;
+  max-width: 700px;
+  padding: 0 30px;
   text-align: left;
-  margin-top: 50px;
+}
+
+.shoppin-cart-image {
+  height: auto;
+  max-width: 100%;
+  padding: 26px 110px;
 }
 
 .welcome-text {
-  margin: 40px 0;
+  margin: 32px 0;
+  font-size: 18px;
+  text-align: center;
 }
 
 .welcome-header {
+  align-items: flex-start;
+  font-weight: 600;
+  font-size: 30px;
+  height: 100px;
   position: relative;
+  text-align: center;
   top: unset;
 }
 
 .center {
   text-align: center;
 }
+
+/** component join-list **/
+.share-token {
+  font-size: 18px;
+  font-weight: 600;
+  width: 100%;
+}
+
+@media (min-width: 1024px) {
+  .welcome-container {
+    justify-content: center;
+    height: 100vh;
+  }
+  .welcome-header {
+    font-size: 38px;
+    padding: 0 24px;
+  }
+
+  .welcome-content {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .shoppin-cart-image {
```

Comment:
```suggestion
  .shopping-cart-image
```

You know it! hahaha 😄

Reply:
Personal opinion: take care with your illustration or image when the screen is smaller.  It's fine if it changes your size in a dynamic way but it's so small it's too. As only a suggestion, think about "Don't display your image if it is so small".

![image](https://user-images.githubusercontent.com/8689897/142734806-1d79d3fc-96ae-4c40-8367-765539f97429.png)

Reply:
Your suggestion is absolutely right, I already did the changes to hide the image with smaller screens